### PR TITLE
0.7.5

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update from 0.7.4 to 0.7.5
- Remove noarch
- Update metadata

https://github.com/dylan-profiler/visions

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4f66c1da369cdbced5e1e2d2c43104caa8cfb9c694006cea1592e1673cfc1153
+  sha256: 49259f291f7737df81c4bc65e9436c43f701acb73a577393be75b26b738d5bd2
 
 build:
   number: 0
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:14]: Skip `py<36`, see https://github.com/dylan-profiler/visions/blob/dca86aa1f6be9d5872f7cce91414e1fa33bd1b85/setup.py#L58

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:17]: Could you build it with `multimethod >=1.4`? See https://github.com/dylan-profiler/visions/blob/dca86aa1f6be9d5872f7cce91414e1fa33bd1b85/requirements.txt#L6

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:14]: added, thank you

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml:17]: corrected, thank you

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**